spec/features/admin/rebate_forms/edit_spec.rb

```diff
@@ -28,6 +28,22 @@
       end
     end
 
+    describe 'condtional render of fields' do
+      it 'can hide moved within rating year and previous year information' do
+        visit "/admin/rebate_forms/#{rebate_form.id}/edit"
+        expect(page).to have_text('Customer details')
+        expect(page).to have_text('Name')
+        choose('fields.lived_in_property_1_July', option: 'yes')
+        expect(page).to_not have_text('Moved within rating year')
+        choose('fields.lived_in_property_1_July', option: 'no')
+        expect(page).to have_text('Moved within rating year')
+        choose('fields.moved_within_rating_year', option: 'no')
+        expect(page).not_to have_text('Settlement date')
+        choose('fields.moved_within_rating_year', option: 'yes')
+        expect(page).to have_text('Settlement date')
+      end
+    end
```

Comment:
I would rewrite it a little for clarity:
```
describe 'condtional render of fields' do
  it 'can hide moved within rating year and previous year information' do
    visit "/admin/rebate_forms/#{rebate_form.id}/edit"
    expect(page).to have_text('Customer details')
    expect(page).to have_text('Name')
    choose('fields.lived_in_property_1_July', option: 'yes')
    expect(page).to_not have_text('Moved within rating year')
    expect(page).to have_text('Settlement date')
    choose('fields.lived_in_property_1_July', option: 'no')
    expect(page).to have_text('Moved within rating year')
    expect(page).not_to have_text('Settlement date')
  end
end
```

Reply:
hmmm this doesn't pass for me - tried to simplify it as much as possible does rspec pass for you with :arrow_double_up: ?

Reply:
Give me one sec ...

Reply:
Nope, you were totally right as is, I misread it. Sorry! Approving now.